lib/blockchain/common.js

```diff
@@ -97,7 +118,9 @@ function fetchAndInstall (coinRec) {
   const binDir = binaries.dir
 
   es(`wget -q ${url}`)
-  es(`tar -xzf ${downloadFile}`)
+  coinRec.cryptoCode === 'XMR'
+    ? es(`tar -xf ${downloadFile}`)
+    : es(`tar -xzf ${downloadFile}`)
```

Comment:
It doesn't feel right to have this XMR condition inside a 'common' file. The alternative here is to check whether we have a bzip2 or gzip and use the appropriate option. 

```suggestion
es(`file ${downloadFile} | awk '{print $2}'`) === 'bzip2'
  ? es(`tar -xf ${downloadFile}`)
  :  es(`tar -xzf ${downloadFile}`)
```

Reply:
Probably it's even worth to have the bzip2 and gzip as a key-value inside the BINARIES object

Reply:
Had to make some slight changes as that command responds with stuff like `Zip` for `.zip`, `Debian` for `.deb`, etc.

Used `echo ${downloadFile} | awk -F. '{print $NF}'`, which will retrieve `gz` or `bz2`, depending on the file. The key/value would be great if we had more than 2 different file extensions being used. As we currently only have XMR using a different extension, I'd believe `.gz` is the most common one